src/components/Stats.tsx

```diff
@@ -50,6 +52,17 @@ export const Stats = (props: {
     return null;
   }
 
+  const version = getVersion();
+  let hash;
+  let timestamp;
+
+  if (version.length > 16) {
+    timestamp = version.slice(0, 16).replace("T", " ");
+    hash = version.slice(21);
+  } else {
+    timestamp = "Version data not available";
```

Comment:
didn't use i18n here (and in the toast) coz it seems unnecessary — this is mostly for devs (and reporters who already communicate with us in English)